src/microbe_stage/Microbe.cs

```diff
@@ -998,6 +1044,10 @@ public override void _Process(float delta)
             totalMovement += queuedMovementForce;
 
             ApplyMovementImpulse(totalMovement, delta);
+
+            // Play movement sound if one isn't already playing.
+            if (!movementAudio.Playing)
+                movementAudio.Play();
```

Comment:
It would be really nice to scale the movement sound level by the cell speed so that only the fastest cells play the sound at max volume. Though if it is too hard to do then this can be left as is and fixed in a future PR.

Reply:
I was trying to set it up for that but the fact I couldn't get the local volume to change sorta made that a no go right now. I was gonna look into it again and try to fix that in a future PR, we might want new/different sounds for cell movement anyway, there's two other already but I didn't actually know what they were supposed to be used for. I looked into changing the playback rate instead of the volume too but godot doesn't really support that either.. you can only play with the pitch to sort of simulate it.